labs/11.md

```diff
@@ -1,8 +1,8 @@
 ---
 title: Lab 11 - Bonus Lab!
 layout: lab
-facilitator: null
-nav_exclude: true
+facilitator: kian, rjz
+nav_exclude: false
```

Comment:
When you do this, it appears on the sidebar, which we don't want

Reply:
oops sorry lemme fix